e2e-tests/nextjs/sentry.edge.config.ts

```diff
@@ -12,5 +12,5 @@ Sentry.init({
 
   // Setting this option to true will print useful information to the console while you're setting up Sentry.
   debug: false,
-  spotlight: process.env.NODE_ENV === 'development',
+  spotlight: true,
```

Comment:
same question generally here: If this is always true, the integration will also try to send events to the side car in production. Is this what we want?

Reply:
I never considered these test codes as representative examples for usage (as we also set `debug: true` on some places). I removed the conditionals mostly to avoid any confusing test issues in case we mess with `NODE_ENV` value for some reason (builds, installing dependencies etc).

For representative code examples, docs should be the main source of truth and yes ideally, Spotlight should not be included in production code but if one wants to do it, we do have a warning somewhere in the code :)

Reply:
Ok I see, that makes sense! Fwiw, at least when I worked on Spotlight the e2e tests were quite messy and probably didn't always reflect what we instruct users to do. I think generally, it's good to stay close to how we instruct the setup but honestly, no strong feelings for this one here. 

Just wanted to make sure we're aware of the bundling/tree-shaking checks around prod vs dev.